packages/nc-gui/composables/useMultiSelect/index.ts

```diff
@@ -106,6 +123,43 @@ export function useMultiSelect(
           if (typeof textToCopy === 'object') {
             textToCopy = JSON.stringify(textToCopy)
           }
+
+          if (columnObj.uidt === UITypes.Formula) {
+            textToCopy = textToCopy.replace(/\b(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}\+\d{2}:\d{2})\b/g, (d: string) => {
```

Comment:
What about a negative timezone? Hope `\+` should be `[+-]`

Reply:
Good catch. updated.